roles/rnaseq/tasks/main.yml

```diff
@@ -38,9 +38,9 @@
   lineinfile:
     dest: "{{ ngi_pipeline_conf }}/{{ item.script }}"
     line: >
-          alias rnaseq='nextflow run {{ rnaseq_dest }}/main.nf
-          -c {{ ngi_pipeline_conf }}/nextflow_irma_{{ item.site }}.config
-          -c {{ ngi_pipeline_conf }}/rnaseq_{{ item.site }}.config'
+          alias rnaseq='nextflow run {{ rnaseq_dest }}/
+              -c {{ ngi_pipeline_conf }}/nextflow_irma_{{ item.site }}.config
```

Comment:
..and here

Reply:
There is a new pull request for them here https://github.com/NationalGenomicsInfrastructure/irma-provision/pull/331